server/server_test.go

```diff
@@ -161,10 +161,11 @@ func (s *ServerURLSuite) TestQRCodeGeneration() {
 	}
 
 	_ = rootCAs.AppendCertsFromPEM(certPem)
+	//nolint: gosec
 	tr := &http.Transport{
 		TLSClientConfig: &tls.Config{
 			MinVersion:         tls.VersionTLS12,
-			InsecureSkipVerify: true, //nolint:all // MUST BE FALSE
+			InsecureSkipVerify: true,
 			RootCAs:            rootCAs,
 		},
```

Comment:
I know you didn't introduce this, but would you be ok removing `InsecureSkipVerify: true,`? 😆 it is hilarious that that the comment says `MUST BE FALSE`, but the implementation sets the value to `true` and `nolint:all` to stop the linter complaining 😭 .

Reply:
sure